src/commands/start_project.rs

```diff
@@ -0,0 +1,56 @@
+use std::fs;
+use std::env;
+use std::fs::File;
+use std::io::prelude::*;
+
```

Comment:
We do not use `use` statement in FifthTry, look at other files. Always use full name of anything. I was looking at line number 15, say `File::create` and I did not know if it is `std::fs::File` or `tokio::fs::File`, I had to jump to the top, and this time I got luck as this is a new file and the use statement was in the PR diff. In other PRs, I would have to click on view file to see what you meant when you said `File`. 

Code is written once, and read thousands of times. Do not optimise for writer, optimise for reader.